docs/source/parallelism.mdx

```diff
@@ -288,11 +293,13 @@ Implementations:
 - [Megatron-LM](https://github.com/NVIDIA/Megatron-LM)
 - [Varuna](https://github.com/microsoft/varuna)
 - [SageMaker](https://arxiv.org/abs/2111.05972)
+- [OSLO](https://github.com/tunib-ai/oslo)
 
-🤗 Transformers status: not yet implemented, since we have no PP and TP.
+🤗 Transformers status:
+- [OSLO](https://github.com/tunib-ai/oslo) is under discussion about integration with the Transformers.
 
 
-## DP+PP+TP+ZeRO
+## ZeRO DP+PP+TP
```

Comment:
Since we introduced the name as ZeRO DP in the upper session, it would be better to write DP and ZeRO as the same rather than separately.